react/SlideToggle/SlideToggle.test.js

```diff
@@ -39,12 +39,12 @@ describe('Slide toggle:', () => {
     const slideToggle = renderIntoDocument(
       <SlideToggle
         id="testToggle"
-        text="Test toggle"
+        label="Test toggle"
         onToggle={onToggle}
       />
     );
     const input = findRenderedDOMComponentWithTag(slideToggle, 'input');
     Simulate.change(input);
-    expect(onToggle).toHaveBeenCalledWith(false);
+    expect(onToggle).toHaveBeenCalled();
```

Comment:
Why are we no longer asserting that the value is being provided to the callback?